assets/app/views/AddSiteView.js

```diff
@@ -0,0 +1,41 @@
+var fs = require('fs');
+
+var Backbone = require('backbone');
+var _ = require('underscore');
+
+var SiteModel = require('../models/Site').model;
+var templateHtml = fs.readFileSync(__dirname + '/../templates/AddSiteTemplate.html').toString();
```

Comment:
Cool, I'll take a look at stringify but the current way of doing it is fine for me as well.